plugin.xml

```diff
@@ -23,8 +25,7 @@
     <source-file src="src/android/camera_theme.xml" target-dir="res/values" />
     <source-file src="src/android/camera_ids.xml" target-dir="res/values" />
 
-    <preference name="ANDROID_SUPPORT_LIBRARY_VERSION" default="26+"/>
-    <framework src="com.android.support:exifinterface:$ANDROID_SUPPORT_LIBRARY_VERSION" />
+    <framework src="com.android.support:exifinterface:25+" />
```

Comment:
Please remove this change for exifInterface version and revert to previous implementation.